tools/memtest.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+
+# Part of anyexec2c project
+# https://github.com/exyi/anyexec2C/tree/master/tools
+#
+# Tests memory limits of the sandbox, measured in blocks of BLOCK_SIZE
+# Exit code 254 probably means that the limit has not been reached when that many blocks were allocated
+# Exit code 255 is some error
+
+import sys, os
+import time
+
+BLOCK_SIZE = 1024*1024
+
+def child(pipe_write):
+	for aloc in range(0, 254):
+		aloc * BLOCK_SIZE * b'*'
```

Comment:
Hmm, I guess we can rely on Python interpreter not getting smart and optimizing this away :shrug:

Reply:
Yes, we're talking about CPython...